src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -938,24 +1024,33 @@ public void deleteTarget(long targetId) {
     complete(submit(req("DELETE", tmpl("/targets/{id}.json").set("id", targetId)), handleStatus()));
   }
 
+  public Iterable<Trigger> getTriggers(Map<String, List<String>> parameters) {
+    return new PagedIterable<>(
+        cbp("/triggers.json", map2ParamList(parameters)), handleList(Trigger.class, "triggers"));
+  }
+
   public Iterable<Trigger> getTriggers() {
-    return new PagedIterable<>(cnst("/triggers.json"), handleList(Trigger.class, "triggers"));
+    return getTriggers(Collections.emptyMap());
   }
 
   public Iterable<Trigger> getTriggers(
       String categoryId, boolean active, String sortBy, SortOrder sortOrder) {
-    return new PagedIterable<>(
-        tmpl("/triggers.json{?category_id,active,sort_by,sort_order}")
-            .set("category_id", categoryId)
-            .set("active", active)
-            .set("sort_by", sortBy)
-            .set("sort_order", sortOrder.getQueryParameter()),
-        handleList(Trigger.class, "triggers"));
+    Map<String, List<String>> parameters = new HashMap<>();
```

Comment:
Another reason why I'm not certain we should allow to pass any parameter, it adds additional logic to the existing code.